docs/config-file-ort-yml.md

```diff
@@ -156,6 +156,25 @@ For details of the specification, see
 The list of available options for `reason` are defined in
 [LicenseFindingCurationReason.kt](../model/src/main/kotlin/config/LicenseFindingCurationReason.kt).
 
+### Curating Third-Party Metadata
+
+Project-specific package curations can be added if you want to correct metadata of third-party dependencies.
```

Comment:
Instead of introducing multiple sections speaking about "project specific" things, how about explaining that once in the first section of this file. E.g. that all configuration in this file applies only to this Project's context. Usually global is preferred for increased degree of automation and local should only be done if there is a good reason.

Reply:
Done in a separate commit

Reply:
Thx!